### ISSUE TITLE
[release-2.2] - Update the Chart appVersion number for all NOTES.tpl files

### ISSUE THREAD

**Issue by chrisbsmith** [DecipherNow/helm-charts#542]
With the installation of the release-2.2 branch of the Helm Charts, the versions represented in the charts.yaml file are not accurate, and so the print out to customers is incorrect.

Review all chart.yaml file and ensure the `{{ Chart.AppVersion}}` is correct

https://github.com/DecipherNow/helm-charts/blob/1074936fbce23860e960b056cec74651ff2862e7/edge/templates/NOTES.txt#L1

**Comment by hiromis** [comment]
@chrisbsmith ,

Just to clarify, do we want `appVersion` to be 2.2.0 and `version` to be the release image tag of each project (unless it's the umbrella chart)?

**Comment by chrisbsmith** [comment]
Yes, the `appVersion` represents the version of the software package.  and `version` represents the version of the helm chart itself; which changes frequently.

**Comment by hiromis** [comment]
> the `appVersion` represents the version of the software package

I am still not 100% clear on what "software package version" means. Please let me know if PR546 does not have the changes that are needed.
